sympy/polys/domains/algebraicfield.py

```diff
@@ -124,3 +126,8 @@ def numer(self, a):
     def denom(self, a):
         """Returns denominator of ``a``. """
         return self.one
+
+    def from_AlgebraicField(K1, a, K0):
+        """Convert a polynomial to ``dtype``. """
```

Comment:
Technically, `a` is represented by a polynomial but it is less confusing to say that it is an algebraic field element. So this should say something about converting an element `a` from one algebraic field to another.

Reply:
`Convert AlgebraicField element 'a' to another AlgebraicField `
Does it seems ok?